components/Controllers/AuctionPageContent/AuctionPageContent.tsx

```diff
@@ -310,3 +317,65 @@ function FieldsetHeader({
     );
   }
 }
+
+type ExchangeLinksProps = {
+  contractAddress: string;
+  tokenId: string;
+};
+
+function ExchangeLinks({ contractAddress, tokenId }: ExchangeLinksProps) {
+  return (
+    <div className={styles.exchangeLinks}>
+      <div>
```

Comment:
I think you could create these in a loop, like:
```tsx
const sites = ['blur', 'opensea', ...];

return (
  <div> className={styles.exchangeLinks}>
    {sites.map(site => (
        <Link
          href={`${exchangeUrlGenerators[site](
            contractAddress,
            tokenId,
          )}`}
          target="_blank">
          <Image src={`/${site}-gray.svg`} alt="" width={24} height={24} />
        </Link>
    )}
  </div>
)
```
Not sure if each link actually needs to have a div wrapper

Reply:
you could also apply the site name as a class on these, then instead of relying on `:nth-child` you can target the class (makes it less fragile in the event we change ordering)